src/plugins/irc-events/link.js

```diff
@@ -52,7 +52,10 @@ module.exports = function(client, chan, msg) {
 	})).slice(0, 5); // Only preview the first 5 URLs in message to avoid abuse
 
 	msg.previews.forEach((preview) => {
-		fetch(normalizeURL(preview.link), {language: client.language}, function(res, err) {
+		fetch(normalizeURL(preview.link), {
+			accept: "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8",
```

Comment:
I don't think so but do you need to do something similar in L108?

Reply:
Correct, not needed there. It will default to `*/*`.